CONTRIBUTING.md

```diff
@@ -87,3 +87,92 @@ This part of the development documentation should be a full reference of all ava
 #### Cookbook
 
 This should contain specific examples and use cases on how to use and implement the previously mentioned topics.
+
+
+## Some notes about your markdown contribution
+
+* The paths and filenames should be english
+* Consider that all markdown headings are taken over in the page navigation of the documentation
+* Use images by copieng existing or create new ones only
+
+
+## The hugo front matter variables
+
+Every file needs at least the following [hugo front matter](https://gohugo.io/content-management/front-matter/) variables on top of your markdown content:
+
+* `title`: A title for the file (language specific). This entry refletcs the main left navigation of the documentation also.
+* `description`: The content description (language specific).
+* `url`: The url path to build (language specific).
+* `weight`: Used for the sorting order in the left main navigation of the documentaion.
+* `menuTitle` (optional): The optional menuTitle entry reflects an alternative entry in the main left navigation of the documentaion. This is especially helpful if the original title is too long for the navigation.
+
+
+## Workflow examples to contribute to the Contao docs repo.
+
+
+### GitHub only workflow 
+
+All you need to contribute to the documentation is a GitHub account. There are no further installations necessary. This approach is handy for changing existing content, for example to change spelling mistakes. If you want to create new content we recommend the "Local Workflow".
+
+First step is to [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the [Contao docs repo](https://github.com/contao/docs) (recommended). As an alternative, you could also edit the markdown files directly in the [Contao docs repo](https://github.com/contao/docs) and save your changes with a comment. GitHub will then automatically create a fork and a new branch with your changes in your repository (named "patch-1" for example).
```

Comment:
You can also use the edit link ... you don't need to create a fork yourself.

Reply:
This workflow is already mentioned as an alternative approach ...

Reply:
Ok, see the latest commit